### DISCUSSION TITLE
滑稽彩快乐！！！

### DISCUSSION THREAD

**Discussion by Enigma-Soul** [Meloong-Git/PCL#1539] [讨论]
![image](https://user-images.githubusercontent.com/119654171/229262789-c8e98c98-0853-40b1-96f1-f6daaab7b9ea.png)

**Comment by WForst-Breeze** [comment]
恭喜~

**Reply by iPodtouch6557** [reply]
surface触屏可以直接解锁

**Comment by iPodtouch6557** [comment]
啊，我是玩个三分钟就有了

**Reply by iPodtouch6557** [reply]
但是恭喜
